upgrade_paths/1.3.0_1.4.0/after/001-acls.postgresql.json

```diff
@@ -0,0 +1,6 @@
+[
+  [ "tags were added for ACLs",
```

Comment:
This test (and the other similar ones) are incomplete. They test that the field `tags` was added, but the migrations do more than adding the field. They also add indexes and triggers.

The index could be tested with a postgresql statement, similar to the one used here, over the `pg_indexes` view.

For testing the trigger, what I would do is test the tags feature itself: create an `ACL` tagged with a unique tag(`acl_foo`) via the Admin API, and then check that the ACL instance appears when filtering by tags in `/acls?tags=acl_foo`. This last step should work in both Cassandra and PostgreSQL, so it would need to be in a separate file.

Reply:
Added the missing tests, but all of them using only db commands.